source/release-notes.txt

```diff
@@ -1354,7 +1354,7 @@ New Features:
 - Improved |compass| startup time.
 
 - Adds support for `MongoDB 5.1 features
-  <https://www.mongodb.com/docs/rapid/release-notes/5.1/>`__.
+  <https://www.mongodb.com/docs/rapid/release-notes/6.0/>`__.
```

Comment:
[suggestion]

I think it's a bit confusing to have folks click 5.1 and land on 6.0 RN. I know 5.1 RN aren't available without going to legacy docs. I would consider unlinking.